Source/Models/HYPFormTarget.m

```diff
@@ -274,6 +292,7 @@ + (void)filteredTargets:(NSArray*)targets
             case HYPFormTargetActionHide:
                 if (![hidden containsObject:target]) [hidden addObject:target];
                 break;
+            case HYPFormTargetActionClear:
```

Comment:
Yeah, it should update the target with a nil value.

Have a look at https://github.com/hyperoslo/HYPForms/blob/feature/add-clear-target/Source/HYPFormsManager.m#L627

I thought it would be smart to share the logic with update instead of duplicating some of the code just to support clearing a field.